games/Secret of Evermore.yaml

```
@@ -27,8 +27,11 @@ Secret of Evermore:
     on: 50
     full: 50
   sniffamizer: # Shuffles or randomizes drops in sniff locations
-    on: 50
-    full: 50
+    shuffle: 75
+    everywhere: 25
+  sniff_ingredients:  # Randomizes the item pool for sniff ingredients
+    vanilla_ingredients: 50
+    random_ingredients: 50
```

I don't think the new option should exist in fillers since it ~triples the total location count.

The following change would basically restore the old behaviour: 50% chance to use shuffled vanilla, 50% chance to use fully random ingredients
```suggestion
    shuffle: 100
  sniff_ingredients:  # Randomizes the item pool for sniff ingredients
    vanilla_ingredients: 50
    random_ingredients: 50
```